### PR TITLE
refactor: Rename GraphQL query methods

### PR DESCRIPTION
**Pull request by zharinov**
Ref #6686

### CODE REVIEW

File: lib/util/http/github.ts [hunk 1]
```diff
@@ -223,14 +223,18 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
...
-    accept = 'application/vnd.github.merge-info-preview+json'
-  ): Promise<GithubGraphqlResponse<T>> {
+    options: GraphqlOptions = {}
+  ): Promise<T> {
     let result = null;
 
     const path = 'graphql';
 
+    const {
+      acceptHeader: accept = 'application/vnd.github.merge-info-preview+json',
```
Comment (rarkins) [inline lib/util/http/github.ts:235]: Isn't it a mistake to have it in this file? Planning to correct it in a follow-up PR?

File: lib/platform/github/index.ts [hunk 2]
```diff
@@ -619,11 +617,9 @@ async function getOpenPrs(): Promise<PrList> {
         }
       }
       `;
-      const nodes = await githubApi.getGraphqlNodes<any>(
-        query,
-        'pullRequests',
-        { paginate: false }
-      );
+      const nodes = await githubApi.queryRepoField<any>(query, 'pullRequests', {
+        paginate: false,
```
Comment (rarkins) [inline lib/platform/github/index.ts:621]: FYI I'm surprised to see pagination false for these

**Review by rarkins — COMMENTED** [review]
Would it be possible to instead combine these into a single `graphql` function with optional `fieldName` param to distinguish between them?

File: lib/util/http/github.ts (hunk 1 above)
Comment (zharinov) [reply lib/util/http/github.ts:235]: Right, after `initRepo` refactoring PR

File: lib/platform/github/index.ts (hunk 2 above)
Comment (zharinov) [reply lib/platform/github/index.ts:621]: I'll enable it in separate PR then

File: lib/util/http/github.ts (hunk 1 above)
Comment (rarkins) [reply lib/util/http/github.ts:235]: If it requires a fix or change to how headers are handled in this graphql, can it be done before?

File: lib/platform/github/index.ts (hunk 2 above)
Comment (rarkins) [reply lib/platform/github/index.ts:621]: I'm trying to think why it was left with no pagination though

**Pull request merged**